README.md

```diff
@@ -78,27 +176,29 @@ So here are three default placeholders for your file system:
 
 - `~` Your **home** directory
 - `.` The **current** directory
-- `..` The directory in which your current directory is contained—referred to as the "**parent**"
-directory.
+- `..` The directory in which your current directory is contained—referred to
+as the "**parent**" directory.
 
 You can supply any path to the `cd` command to nkellyegreenegate to that location.
 
 ## Demonstrate Manipulating Files in the Command Line
 
 ### Paths in Shell
 
-The path supplied to the `cd` command, for example `/Users/kellyegreene`, is known as an absolute
-path.
+The path supplied to the `cd` command, for example `/Users/kellyegreene`, is
+known as an absolute path.
 
 Systems can use either *absolute* or *relative* paths.
 
-An absolute path is a path that points to the same location on the file system regardless of the
-working directory. They start with `/` ("forward slash") because that is the root of your file system.
+An absolute path is a path that points to the same location on the file system
+regardless of the working directory. They start with `/` ("forward slash") because
+that is the root of your file system.
 
 This is an absolute path: `/Users/kellyegreene`.
 
-A relative path is a path **relative** to the working directory of the user or application, so the
-full absolute path will not have to be given. They start with the name of a directory or a file.
+A relative path is a path **relative** to the working directory of the user or
+application, so the full absolute path will not have to be given. They start with
+the name of a directory or a file.
 
 This is a relative path: `kellyegreene/Documents`.
 
```

Comment:
If I were in my home directory `/Users/kellyegreene` directory and said `cd mixtapes/the-masked-rapper-vol-1`, it would work! If I were in `/Users/annoyingbrother` and said `cd mixtapes/the-masked-rapper-vol-1`, `bash` would return an error because that sub-directory doesn't exist there.

Relative paths exist to make it simpler to move around. Real life is like this too. If someone asks if you want to go get a slice of pizza they're probably thinking within a few blocks or miles of where you're currently located – somewhere close _relatively_ speaking.

"Oh yeah, let's go west on 18th street, cross 6th avenue and go to the place on the corner." 

What would surprise the heck out of them would be if you gave them absolute coordinates; and it would be _even more_ surprising if that location were far away:

"Oh yeah, let's go to 41.890221 and -87.633904!"